addons/account/wizard/account_move_reversal.py

```diff
@@ -112,6 +112,7 @@ def _prepare_default_reversal(self, move):
     def reverse_moves(self):
         self.ensure_one()
         moves = self.move_ids
+        self.refund_method = self._context.get('default_refund_method') or self.refund_method
```

Comment:
do you need to check the context? the point of passing the key default_refund_method is that it will give proper default value at the record creation :point_up:

Reply:
I just thought: maybe it's because of the override of default_get that my previous suggestion doesn't work